charts/ingress-nginx/README.md.gotmpl

```diff
@@ -226,4 +226,24 @@ Detail of how and why are in [this issue](https://github.com/helm/charts/pull/13
 
 As of version `1.26.0` of this chart, by simply not providing any clusterIP value, `invalid: spec.clusterIP: Invalid value: "": field is immutable` will no longer occur since `clusterIP: ""` will not be rendered.
 
+### Pod Security Admission
+
+You can use PSA by applying label to `ingress-nginx` namespace as instructed by [Pod Security Admission](https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-namespace-labels/)
+
+For now, highest level can be <strong>enforced</strong> is `baseline`
```

Comment:
```suggestion
For now, highest level, that can be enforced, is `baseline`
```

I'm not sure about this. We put a lot of effort into making it more secure in the past, so we should also comply to `restricted`. Please test this and tell why you think it would not work. Thank you!

Reply:
The reason is I read from this PR https://github.com/kubernetes/ingress-nginx/pull/10608#issue-1975114429

Sure, I will have a test with `restricted`

Reply:
This PR is outdated. There are a lot of other PRs contributing to making the chart more secure. I'm actually running it in an environment that's enforcing `restricted` for a while now, but please feel free to validate this.

Reply:
Thanks for that. I just did a test, no issue with `restricted`